### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN013.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#59] [General]
# https://blog.3place.cn/3place/lmyn/LMYN013.html

https://blog.3place.cn/3place/lmyn/LMYN013.html

**Comment by picklecai** [comment]
工程量真浩大！还要搞新建筑，邀请社会名流。真像奥运会。

**Comment by picklecai** [comment]
各个种姓都会受到
应有的尊敬和款待。 
不能因为偏爱或发怒，
就表现出种种歧视。

本意是没有种姓偏爱。
